zerver/migrations/0455_alter_scheduledmessagenotificationemail_trigger.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.1 on 2023-06-02 05:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0454_alter_scheduledmessagenotificationemail_trigger"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="scheduledmessagenotificationemail",
+            name="trigger",
+            field=models.TextField(
+                choices=[
+                    ("private_message", "Private message"),
+                    ("mentioned", "Mention"),
+                    ("wildcard_mentioned", "Wildcard mention"),
+                    ("stream_email_notify", "Stream notifications enabled"),
+                    ("followed_topic_email_notify", "Followed Topic notifications enabled"),
+                    ("followed_topic_wildcard_mentioned", "Followed Topic wildcard mention"),
```

Comment:
This migration should be dropped in favor of adding the new fields to this existing migration:
`zerver/migrations/0331_scheduledmessagenotificationemail.py`

As noted before, this is not actually a database change, it's just a change to Django's Python code understanding of what values are possible, and it's a distraction to treat it as an independent migration.

Reply:
Pushed this in my revisions to this PR.